app/src/main/java/ch/epfl/sdp/entity/PlayerManager.java

```diff
@@ -15,7 +15,7 @@
  * beginning of each game and all players should be removed at the end of each game.
  */
 public class PlayerManager {
-    public static final int NUMBER_OF_PLAYERS_IN_LOBBY = 2;
+    public static final int NUMBER_OF_PLAYERS_IN_LOBBY = 1;
```

Comment:
Was this done for testing? A lobby with one player seems a bit weird to me. And now we have the solo mode, so it should also work fine (maybe you forgot to remove this)